python/lsst/faro/measurement/TractTableMeasurementTasks.py

```diff
@@ -144,3 +150,123 @@ def run(
                 metricName, np.mean(np.abs(result["corr"])), extras=extras
             )
         )
+
+
+class WPerpTableConfig(MeasurementTaskConfig):
+    """Class to organize the yaml configuration parameters to be passed to
```

Comment:
Should this say TExTableTask?

Reply:
Why do you need a different class for wPerp, xPerp etc, also doesn't this also calculate wPara as well?

Reply:
Because an lsst.verify.Measurement object must be a single scalar value, each measurement task must always be separate. We are considering a way of persisting the calculated values so that related metrics can simply do the calculation once and read/use the values for multiple metrics, but that hasn't been implemented yet in `faro`.